client/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export default function Header() {
+  return (
+    <div className="header">
+      <img
+        src="https://codeyourfuture.io/wp-content/uploads/2019/03/cyf_brand.png"
+        alt="cyf-logo"
```

Comment:
The `alt` attribute is important for accessibility reasons - for example this will be read out by most screen reader applications.

For example - in this case the image is showing a logo representing Code Your Future - so `alt="Code Your Future"` could be more informative here